src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -218,91 +254,102 @@ def probsolve_ivp(
      [0.97]
      [0.98]
      [0.99]]
+
     """
-    stprl = _create_steprule(atol, rtol, step, firststep, ivp)
-    prior = _string2prior(ivp, which_prior, **kwargs)
-    gfilt = _create_filter(ivp, prior, method, **kwargs)
-    with_smoothing = method[-2] == "s" or method[-1] == "s"
-    solver = GaussianIVPFilter(ivp, gfilt, with_smoothing=with_smoothing)
+
+    # Normalize string inputs
+    method = method.upper()
+    which_prior = which_prior.upper()
+
+    # Create IVP object
+    ivp = IVP(timespan=(t0, tmax), initrv=pnrv.Constant(np.asarray(y0)), rhs=f, jac=df)
+
+    # Infer first step from arguments
+    if step is None:
+        # lazy version of Hairer, Wanner, Norsett, p. 169
+        norm_y0 = np.linalg.norm(ivp.initrv.mean)
+        norm_dy0 = np.linalg.norm(ivp(ivp.t0, ivp.initrv.mean))
+        firststep = 0.01 * norm_y0 / norm_dy0
+    else:
+        firststep = step
+
+    # Set up some default hyperparameters: not great, but much better than 1.0
+    lengthscale = lengthscale if lengthscale is not None else 1.0 / firststep
+    driftspeed = driftspeed if driftspeed is not None else firststep
```

Comment:
turns out that these are much better than choosing 1.0 (see the adaptive stepsize notebook). Of course they are not perfect, but it works surprisingly well.